src/components/App.tsx

```diff
@@ -27,9 +47,72 @@ const App: React.FC = () => {
           <Grid.Column stretched width={12}>
             <Segment>
               <Switch>
-                <Route path="/" exact component={Home} />
-                <Route path="/form" component={Register} />
-                <Route path="/applications" component={ApplicationsList} />
+                <Route exact path="/">
+                  <Home />
```

Comment:
Just by curiosity: Why not use the `component={Home}` pattern?

Reply:
Good question! I noticed the [ReactRouter docs](https://reactrouter.com/web/example/basic) used this style throughout, so I asked on SSNZDevs what the difference was (if any). There isn't really any, but both Josh and Will said this way is the preferred way these days, as it's more "declaritive" which is in keeping with general React style. 

Also, the other way gets really ugly once you start having to pass props to the components. ;)